CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 
 ### Breaking Changes
 
-_None_
+* Update the API of `ios_merge_strings_files` and `ios_extract_keys_from_strings_files` to support using prefixes for string keys when merging/splitting the files.
+  The actions now expect a `Hash` (instead of an `Array`) for the list of files to provide an associated prefix (or `nil` or `''` when none) for each file to merge/split. [#345]
```

Comment:
This is a breaking change as I didn't feel it would be worth the effort to make sure it was backwards-compatible, given it's not used by any client app yet.

That being said, still allowing the action to also accept an `Array` for cases where you don't need to use prefixes for any of the files being merged/extracted (like it might be the case for WCiOS…) could make it for a nicer call site for those special cases (using `[file1, file2]` rather than `{file1 => nil, file2 => nil}` when you don't want any prefix)… 🤔 🤷 

See also https://github.com/wordpress-mobile/release-toolkit/pull/345/files#r822930611